documentation/source/overview/studies.rst

```diff
@@ -5,96 +5,96 @@ Studies using SCT
 
```

Comment:
Small heads-up: If https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3504 is merged before this PR, those references will need to be reformatted too. 

(Depends on which changes finish first...)

Reply:
Sounds good, @joshuacwnewton